### PR TITLE
added links to special characters and their names

### PR DESCRIPTION
**Pull request by mkruijt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Here you can find course content and homework for the JavaScript 1 module
...
-|2.|• Intro JavaScript (What is it, where can you use it for)<br>• [Variables (var, let, const)](../../../fundamentals/blob/master/fundamentals/variables.md)<br>• [Basic Data types (Strings, Numbers, Arrays, Booleans)](../../../fundamentals/blob/master/fundamentals/values.md)<br>• [Operators](../../../fundamentals/blob/master/fundamentals/operators.md)<br>• [Naming conventions](../../../fundamentals/blob/master/fundamentals/naming_conventions.md)|[Reading Week 2](/Week2/README.md)|[Homework Week 2](/Week2/MAKEME.md)|
+|2.|• Intro JavaScript (What is it, where can you use it for)<br>• [Variables (var, let, const)](../../../fundamentals/blob/master/fundamentals/variables.md)<br>• [Basic Data types (Strings, Numbers, Arrays, Booleans)](../../../fundamentals/blob/master/fundamentals/values.md)<br>• [Operators](../../../fundamentals/blob/master/fundamentals/operators.md)<br>• [Special characters and their names](../../../fundamentals/names_of_special_characters.md)<br>• [Naming conventions](../../../fundamentals/blob/master/fundamentals/naming_conventions.md)|[Reading Week 2](/Week2/README.md)|[Homework Week 2](/Week2/MAKEME.md)|
```
Comment (remarcmij) [inline README.md]: The link doesn't work.

File: Week2/README.md [hunk 2]
```diff
@@ -35,10 +35,12 @@ Only watch the below chapters:
...
-- Intro JavaScript (What is it, where can you use it for)
-- [Variables (var, let, const)](./../../../../fundamentals/blob/master/fundamentals/variables.md)<br>
-- [Basic Data types (Strings, Numbers, Arrays, Booleans)](./../../../../fundamentals/blob/master/fundamentals/values.md)<br>
-- [Operators](./../../../../fundamentals/blob/master/fundamentals/operators.md)<br>
+
+- Intro JavaScript (What is it, where can you use it for)  
+- [Variables (var, let, const)](./../../../../fundamentals/blob/master/fundamentals/variables.md)  
+- [Basic Data types (Strings, Numbers, Arrays, Booleans)](./../../../../fundamentals/blob/master/fundamentals/values.md)  
+- [Operators](./../../../../fundamentals/blob/master/fundamentals/operators.md)  
+- [Special characters and their names](./../../../fundamentals/names_of_special_characters.md)
```
Comment (remarcmij) [inline Week2/README.md]: The link doesn't work.

File: README.md (hunk 1 above)
Comment (remarcmij) [inline README.md]: The link doesn't work.

Comment (remarcmij) [inline README.md]: The link doesn't work.

File: Week3/MAKEME.md [hunk 3]
```diff
@@ -112,7 +113,7 @@ let favoriteAnimals = ['blowfish', 'capricorn', 'giraffe'];
         console.log("yes")
     } else {
         console.log("no")
-    }
+    };
```
Comment (remarcmij) [inline Week3/MAKEME.md]: There are missing colons, but not where you added them 🙃

**Review by remarcmij — CHANGES_REQUESTED** [review]
I think you need to double check the new links. They don't work. Needs extra `/fundamentals/blob/master/` in the URLs

File: Week3/MAKEME.md (hunk 3 above)
Comment (mkruijt) [reply Week3/MAKEME.md]: ha yes, you are right

File: README.md (hunk 1 above)
Comment (mkruijt) [reply README.md]: ha I sort of did not manage to get one correct, I'm doubting my previous decision to have relative links. It's a mess and does not make sense. But maybe this should be a worry for tomorrow.

File: README.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ Here you can find course content and homework for the JavaScript 1 module
...
-|2.|• Intro JavaScript (What is it, where can you use it for)<br>• [Variables (var, let, const)](../../../fundamentals/blob/master/fundamentals/variables.md)<br>• [Basic Data types (Strings, Numbers, Arrays, Booleans)](../../../fundamentals/blob/master/fundamentals/values.md)<br>• [Operators](../../../fundamentals/blob/master/fundamentals/operators.md)<br>• [Naming conventions](../../../fundamentals/blob/master/fundamentals/naming_conventions.md)|[Reading Week 2](/Week2/README.md)|[Homework Week 2](/Week2/MAKEME.md)|
+|2.|• Intro JavaScript (What is it, where can you use it for)<br>• [Variables (var, let, const)](../../../fundamentals/blob/master/fundamentals/variables.md)<br>• [Basic Data types (Strings, Numbers, Arrays, Booleans)](../../../fundamentals/blob/master/fundamentals/values.md)<br>• [Operators](../../../fundamentals/blob/master/fundamentals/operators.md)<br>• [Special characters and their names](../../../fundamentals/blob/master/names_of_special_characters.md)<br>• [Naming conventions](../../../fundamentals/blob/master/fundamentals/naming_conventions.md)|[Reading Week 2](/Week2/README.md)|[Homework Week 2](/Week2/MAKEME.md)|
```
Comment (remarcmij) [inline README.md]: Link still doesn't work. Missing extra `fundamentals` in the path.

**Review by remarcmij — CHANGES_REQUESTED** [review]
One broken link remaining...

**Review by remarcmij — APPROVED** [review]
👍

**Pull request merged**